experimental/packages/opentelemetry-sdk-node/README.md

```diff
@@ -102,6 +102,10 @@ Deprecated, please use [logRecordProcessors](#logrecordprocessors) instead.
 
 An array of log record processors to register to the logger provider.
 
+### mergeResourceWithDefaults
+
+Merge user-provided resources with the default resource. Default `true`.
```

Comment:
Could possibly add the note here, as well, about this default changing to `false` in 2.0.

Reply:
yep, good idea 👍 
```suggestion
Merge user-provided resources with the default resource. Default `true`.
The default will change to `false` in a future iteration of this package.
```